frontend/models/mock-search-meta.tsx

```diff
@@ -0,0 +1,63 @@
+import { faCaretRight, faAngleRight, IconDefinition } from "@fortawesome/free-solid-svg-icons"
```

Comment:
Is there a reason that mock-search-meta isn't in `__mocks__`? Also, `__mocks__` is used by Jest to mock modules, rather than to hold mock data. So I would suggest renaming `__mocks__` to mockData, moving all the mock data files there, and then having an `index.ts` file that re-exports everything for easier import.

Reply:
I wanted to make something that would be easy to switch over to using API data once that's working, but also work right away with Jest. However, a drawback of that approach is that I don't really understand how mocks work. Also, we could just put it in `models/` and then change one line of code when there's another source. But someday it would be good to have a mock data source for testing, if somebody knows how to do that.

Reply:
The mock-search-meta is a separate file in case the column names aren't available from the API

Reply:
Got it, I don't think we'll get column names from the API so consider renaming this to just search-meta.

I'm planning on adding [mock-service-worker](https://mswjs.io/) in #152 which will let us mock the API endpoints directly, and we can pipe in the mock data there. We can just leave the `__mocks__` as is as we'll need to rejigger things for that.